app/views/shared/_recipe_show.html.erb

```diff
@@ -0,0 +1,67 @@
+<section class="container">
+  <h1 class="text-center mb-5"><%= recipe.name %></h1>
+  <span class="badge badge-primary">Private</span>
+  <div class="recipe-info d-flex justify-content-around">
+    <div>
+      <h5>Preparation time: <%= recipe.preparation_time %></h5>
+      <h5>Cooking time: <%= recipe.cooking_time %></h5>
+      <div>
+        <h3 class="mt-5">Description</h3>
+        <p><%= recipe.description %></p>
+      </div>
+    </div>
+    <div class="recipe-stat">
+      <% if recipe.public %>
+      <span class="btn btn-warning">Public</span>
+      <% else %>
+      <span class="btn btn-primary">Private</span>
+      <% end %>
+    </div>
+  </div>
+  <div class="mt-5 d-flex justify-content-around">
+    <button
+      type="button"
+      class="btn btn-primary"
+      data-bs-toggle="modal"
+      data-bs-target="#recipeModal"
+    >
+      <%= link_to "Generate shopping list", general_shopping_list_path %>
+    </button>
+    <% if can?(:update, recipe) %> <%= link_to "Add ingredient",
+    recipe_recipe_food_path(recipe), class: "btn btn-danger" %> <% end %>
+  </div>
```

Comment:
when I click on Add ingredient , I get a routing error , Kindly fix this issue 👍 
![image](https://user-images.githubusercontent.com/87448628/197343677-5e51e5bf-2579-47f4-8c67-ba2b983ca7d7.png)